templates.txt

```diff
@@ -44,3 +73,28 @@ hex=HEX..END construct
 = |
 =end // hex
 
+bthold=Hold ingame button template
+=if // Insert the number of the ingame button to hold below
+=	00E1:   player 0 pressed_key |
```

Comment:
this is a game-specific opcode. In LCS/VCS 00E1 has different meaning. 

Can you move this template and `kbhold` to gta3/vc/sa templates?

Reply:
Thanks for this detail! Sure, I'll change it as soon as possible!

Reply:
You can also implement those templates for LCS/VCS using appropriate opcodes

Reply:
I've never worked with LCS and VCS, but I hope I chose the right opcodes for `bthold` template. :)